docs/modules/operation/pages/events/event-definition.adoc

```diff
@@ -0,0 +1,58 @@
+
+[[ga-events-event-definition]]
+= Event Definition
+
+
+== Associate an event to a specific node
+
+There are two ways to associate an existing node to a given event prior to sending it to the event bus:
+
+* Set the event's `nodeId` field to the database ID of the desired node.
+* For requisitioned nodes, set the `_foreignSource` and `_foreignId` as parameters to the event.
+Then, any incoming event without a `nodeId` and these two parameters will trigger a database lookup.
+If a node is found, the `nodeId` attribute will be dynamically set on the event, regardless of which method has been used to send it to the event bus.
+
+
+[[ga-events-anatomy-of-an-event]]
+== Anatomy of an event
+
+Events are structured historical records of things that happen in {page-component-title} and the nodes, interfaces, and services it monitors.
+Every event has a number of fixed fields and zero or more parameters.
+
+.Mandatory Fields
+UEI (Unique Event Identifier)::
+    A string that uniquely identifies the event's type.
+    UEIs are typically formatted in the style of a URI, but the only requirement is that they start with the string `uei.`.
+Event Label::
+    A short, static label that summarizes the gist of all instances of this event.
+Description::
+    A long-form description of all instances of this event.
+    You can use HTML formatting to format the body of the event text.
+Log Message::
+    A long-form log message describing this event, optionally including expansions of fields and parameters so that the value is tailored to the event at hand.
+    The possible options for this setting are the following:
+    logndisplay:::
+        Both log the event in the database and display it in the Web UI.
+    logonly:::
+        Log the event in the database, but do not display it.
+    suppress:::
+        Neither log the event in the database nor display it.
+    donotpersist:::
+        Do not log the event in the database, but still send it to daemons that are listening for this type of event, such as those being used to generate notifications.
+        Note that eventd and alarmd don't process this type of events.
```

Comment:
Do we want to put this disclaimer in an admonition?

Reply:
No, let's minimize admonitions unless they are really serious.